azure-nextgen-ts-webapp-privateendpoint-vnet-injection/index.ts

```diff
@@ -0,0 +1,174 @@
+// Copyright 2016-2021, Pulumi Corporation.  All rights reserved.
+
+import * as azure_nextgen from "@pulumi/azure-nextgen";
+import * as pulumi from "@pulumi/pulumi";
+import * as random from "@pulumi/random";
+
+const config = new pulumi.Config();
+const resourceGroupNameParam = config.require("resourceGroupNameParam");
+
+const locationParam = config.get("locationParam") || "westus2";
+const resourceGroup = new azure_nextgen.resources.latest.ResourceGroup("resourceGroup", {
+    resourceGroupName: resourceGroupNameParam,
+    location: locationParam,
+});
+
+const serverFarmNameParam = config.get("serverFarmNameParam") || "appserverfarm";
+const skuFamilyParam = config.get("skuFamilyParam") || "P1v2";
+const skuNameParam = config.get("skuNameParam") || "P1v2";
+const skuSizeParam = config.get("skuSizeParam") || "P1v2";
```

Comment:
It feels odd to have three parameters for the same thing. I know you are copying the ARM template but I'd take the effort to minimize the code. ARM templates are horrible :)

Reply:
Yup - result of the arm2pulumi conversion. Fixed up now.